tests/fixtures/samples.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import arrow
+
+
+@pytest.fixture
+def mock_sample(static_time):
+
+    return {
+        "_id": "test",
+        "name": "Test",
+        "created_at": static_time.datetime,
+        "ready": False,
+        "files": [
+            {
+                "id": "foo",
+                "name": "Bar.fq.gz",
+                "download_url": "/download/samples/files/file_1.fq.gz",
+            }
+        ],
+        "labels": [],
+        "subtractions": ["foo", "bar"],
+        "user": {"id": ""},
+    }
+
+
+@pytest.fixture
+async def mock_samples(fake, mock_sample, static_time):
+
+    user_1 = await fake.users.insert()
+    user_2 = await fake.users.insert()
+    user_3 = await fake.users.insert()
+
+    return [
+        dict(
+            mock_sample,
+            user={"id": user_1["_id"]},
+        ),
```

Comment:
Prefer `{}` literal over `dict`. It is faster.
```suggestion
        {
            **mock_sample,
            "user": {"id": user_1["_id"]},
        },
```